store/helpers/pages/pages.js

```diff
@@ -0,0 +1,97 @@
+import getOrder from '../order'
+import Page from './page'
+import invalidatePreviousRequests from '../invalidatePreviousRequests'
+
+import { LinkedList } from 'utils/linkedList'
+import apiRequest from 'api'
+import { CancelablePromise } from 'utils/promise'
+
+const PAGE_SIZE = 100
+
+export default class Pages {
```

Comment:
Try always export explicitly in the end of file in a single place. Sometimes you might want to add a decorator to the class and this will affect more lines than it actually should. Also, export in a single point improve readability.